src/base/utils/random.cpp

```diff
@@ -30,25 +30,23 @@
 
 #include <random>
 
+#include <QtLogging>
 #include <QtSystemDetection>
 
 #ifdef Q_OS_WIN
 #include <windows.h>
-#include <ntsecapi.h>
-#else  // Q_OS_WIN
+#include "base/global.h"
+#include "base/utils/os.h"
+#elif defined(Q_OS_LINUX)
```

Comment:
Let's not mix full and short syntax. Could you change `#ifdef Q_OS_WIN` above to `#if defined(Q_OS_WIN)`?

Reply:
Done.